README.md

```diff
@@ -14,19 +16,28 @@ pip install imap-data-access
 imap-data-access -h
 ```
 
+### Base Command Arguments
+```bash
+$ imap-data-access query # or
+$ imap-data-access download # or
+$ imap-data-access upload
+```
+Add the '-h' help flag with any base command for more information
```

Comment:
Great idea to include the bear-bones usage here for quick reference. I think between this and the link to the more detailed instructions is a good balance.